.github/workflows/common.yml

```diff
@@ -35,3 +35,6 @@ jobs:
       - name: Run unit tests
         run: make unit
         working-directory: ${{ env.PACKAGEPATH }}
+      - name: Run configuration validator integration tests
```

Comment:
Rather than adding this to the common CI workflow, can you move the cli to a `validator` directory in the [tools](https://github.com/envoyproxy/xds-relay/tree/master/tools) dir, and create a new GitHub checks workflow that only runs this test in the validator directory?

Reply:
Or are the tests dependent on other packages in the main app directories?

Reply:
good idea.

Reply:
no, the validator integration tests are pretty independent. They build the binary and interact with it directly.